gapis/gfxapi/gfxapi.proto

```diff
@@ -50,6 +50,9 @@ enum ResourceType {
 	ShaderResource = 5;
 	// ProgramResource represents the Program resource type
 	ProgramResource = 6;
+	// UninitializedTextureResource represents texture resource type
+	// which has been generated but does not have dimensions yet.
+	UninitializedTextureResource = 7;
```

Comment:
This isn't a new resource type. Resources exist for the whole trace. Only their data is tied to a single point in time.